src/libraries/Common/src/System/IO/FileSystem.DirectoryCreation.Windows.cs

```diff
@@ -2,13 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.Win32.SafeHandles;
-using System;
+#nullable enable
 using System.Collections.Generic;
-using System.Diagnostics;
 using System.Runtime.InteropServices;
-using System.IO;
```

Comment:
It seems like these namespaces are needed when this file is changed from other projects since are `Common` files. For example if MS_IO_REDIST is defined, then the namespace will be `Microsoft.IO` and so `System.IO` and `System` need to be declared in order to find members under those namespaces.

Reply:
Makes sense, thanks!